lua/entities/gmod_tardis/modules/cl_classic_doors.lua

```diff
@@ -0,0 +1,35 @@
+local function SetVisibleIfExists(part, visible)
```

Comment:
This should ideally be replaced with a ShouldDrawPart hook, I thought one existed but it doesn't, but it should be a pretty trivial change to make in the parts system [around here](https://github.com/MattJeanes/TARDIS/blob/4f2d7937f153faa20537c7777f4e2fa176ba4935/lua/tardis/sh_parts.lua#L12-L18). Spamming `SetRenderMode` / `SetColor` hooks can cause frame drops and here it is not necessary as we have a better and cleaner solution.

Reply:
Should be fixed